Woof/Woof/View/OwnerCardView.swift

```diff
@@ -0,0 +1,71 @@
+import SwiftUI
+
+/// OwnerCardView is a SwiftUI view that displays the profile information in card view style of an Owner.
+struct OwnerCardView: View {
+    @ObservedObject var viewModel = OwnerCardViewModel()
+    @State private var isEditMode: Bool = false
+
+    var body: some View {
+        VStack {
+            HStack {
+                Image(systemName: "person.circle")
+                    .resizable()
+                    .frame(width: 40, height: 40)
```

Comment:
Please, put this sizes in the file **UIElementSize.swift** in the **Helper** directory.